account_multicurrency_revaluation_type/model/res_company.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    rate_type = fields.Selection(
```

Comment:
I would name this field `currency_rate_type` instead, easier to understand what we are talking about in the UI/database schema.